### DISCUSSION TITLE
windows 11arm64

### DISCUSSION THREAD

**Discussion by mjonsson1986** [utmapp/UTM#3178] [UTM Help (macOS)]
seems i can boot a qcow2 image with win 11 arm64 preview
but seems it stays on the first screen whee it says: "getting ready" or something
should it stay there in maybe 2 hours?

**Comment by ghost** [comment]
I'm actually experiencing the exact same problem. This is however my first time using UTM.
<img width="1552" alt="1" src="https://user-images.githubusercontent.com/57814383/136833103-75deda9f-255f-474a-ac4b-f1dfa9c96a99.png">
<img width="912" alt="2" src="https://user-images.githubusercontent.com/57814383/136833108-7db12412-8ca3-4e56-bbf8-a7e0f170e08a.png">

**Comment by conath** [comment]
This has been happening since a recent Windows 11 Insider Preview update, as reported by many on Discord. For the moment it looks like you‘ll need to find a Windows 11 ISO or wait for the next Insider Preview update and try again. 

Since this is not a UTM problem, I’ll move this to the Discussion section.

**Comment by mjonsson1986** [comment]
eh? windows 11 arm isos? exist that?

**Reply by conath** [reply]
Yes, search online (using Windows PC) for UUP Dump website

**Comment by mjonsson1986** [comment]
the only arm release are that vhdx
